src/main/java/net/minecraftforge/client/model/obj/ObjModel.java

```diff
@@ -299,6 +316,10 @@ static ObjModel parse(ObjTokenizer tokenizer, ModelSettings settings, Map<String
                 }
             }
         }
+        if (!allowHomonymObjectName && hasHomonymObjectNameAppeared)
+        {
+            LOGGER.error("find homonym object in obj model:" + settings.modelLocation + ", use allow_homonym_object_name=true can rename them for you");
```

Comment:
Some error message cleanup, as well as using logger parameter substitution instead of manual string concatenation:
```suggestion
            LOGGER.error("Found homonym object in ObjModel: {}; use the 'allow_homonym_object_name' option to automatically resolve homonyms", settings.modelLocation);
```